test/functional/test_site_special_tags/expected/index.html

```diff
@@ -84,6 +76,7 @@ <h2 id="so-far-as-to-comply-with-the-commonmark-spec">So far as to comply with t
             */
           </p>
         </abc>
+        <p></p>
```

Comment:
the two `<p></p>` additions in this file occur due to miswritten markdown as well.
In the first case, the `/* ...` indicates the start of a paragraph ending after the `</abc>`
In the second case, the `if ...` does the same